packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewClientHostApiImpl.java

```diff
@@ -95,6 +98,13 @@ public void doUpdateVisitedHistory(
       flutterApi.doUpdateVisitedHistory(this, view, url, isReload, reply -> {});
     }
 
+    @Override
+    public void onReceivedHttpAuthRequest(
+        @NonNull WebView view, HttpAuthHandler handler, String host, String realm) {
```

Comment:
The FlutterAPI should pass back the `HttpAuthHandler` instead of adding a `setAuthCredentials` method. Mainly because the `HttpAuthHandler` has multiple methods that could be used. e.g. `cancel` and [useHttpAuthUsernamePassword](https://developer.android.com/reference/android/webkit/HttpAuthHandler#useHttpAuthUsernamePassword()).